src/nemos/solvers/_compute_defaults.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Callable, Optional, Tuple, Union
+
+import jax
+
+from ..observation_models import PoissonObservations
+from ..regularizer import Ridge
+from ._svrg_defaults import (
+    glm_softplus_poisson_l_max_and_l,
+    svrg_optimal_batch_and_stepsize,
+)
+
+if TYPE_CHECKING:
+    from ..glm import GLM, PopulationGLM
+
+
+def glm_compute_optimal_stepsize_configs(
+    model: Union[GLM, PopulationGLM]
+) -> Tuple[Optional[Callable], Optional[Callable], Optional[float]]:
+    """
+    Compute configuration functions for optimal step size selection based on the model.
+
+    This function returns a tuple of three elements that are used for configuring the
+    optimal step size and batch size for variance reduced gradient (SVRG and
+    ProxSVRG) algorithms. If the model is configured with specific solver names,
+    the appropriate computation functions are returned. Additionally, it determines the
+    smoothness and strong convexity constants based on the model's observation and regularizer.
+
+    Parameters
+    ----------
+    model :
+        The generalized linear model object for which the optimal step size and batch
+        configuration need to be computed. The model should have attributes like
```

Comment:
"attributes like" or it needs to have those attributes?

Reply:
probably don't need that second sentence at all, the type-hinting above specifies it must be an instance of `GLM` or `PopulationGLM`